### ISSUE TITLE
Bug with git diff replicating

### ISSUE THREAD

**Issue by VB-is-Terrible** [atom/github#1145]
### Prerequisites

* Have you tried to reproduce the problem in [Safe Mode][safe-mode]? Yes
* Have you followed all applicable steps in the [debugging guide][debugging-guide]? Yes
* Have you checked the FAQs on the [message board][message-board] for common solutions? It's a new bug, no solutions exist
* Have you checked that your issue [isn't already filed][already-filed]? Yes

[safe-mode]: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
[debugging-guide]: http://flight-manual.atom.io/hacking-atom/sections/debugging/
[message-board]: https://discuss.atom.io/c/faq
[already-filed]: https://github.com/atom/github/issues

[Discuss topic](https://discuss.atom.io/t/bug-with-git-diff-split-view-in-atom-1-20-beta3-x64/46608)
### Description

The git diff panel gets copied to the active directory on some key strokes. See images.

### Steps to Reproduce
Prerequisites: Have a unstaged/uncommited change in git
1. Have at least two panes open
2. Open git tab open
3. Open a git diff panel in one of the panes
4. Switch to a different
5. [Keyboard] Ctrl+S (Alt-tab also works)

**Expected behavior:**

Nothing

**Actual behavior:**

The git diff panel copies into the active pane

**Reproduces how often:**

100%

### Versions

Atom 1.20-beta3 x64 on Ubuntu 17.04
Atom 1.20.0-beta5 on Ubuntu 16.04

### Additional Information

Before:
![atom-before](https://user-images.githubusercontent.com/12043629/30000786-30b0303c-90bb-11e7-9f0c-67fdb4eeb4de.png)
[Alt-tab out, Alt-tab in]
![atom-after](https://user-images.githubusercontent.com/12043629/30000787-390cc646-90bb-11e7-89d0-4c3e4a29ae7c.png)

No errors in console

There isn't anything abnormal in the keybinding resolver
Crtl+s only contains core:save
Alt-tab isn't recognized by the resolver (Probably intercepted by my compositor)

---

*Some details from @rsese*:

With the Git panel enabled:

1. Open some file in a repo and split the pane right.
2. Save a change to create an unstaged change.
3. Click the unstaged change in the Git panel to open the diff view.
4. Click in the pane that doesn't have the diff view in it.
5. <kbd>alt-tab</kbd> out of Atom and then back into Atom.

The diff view is then duplicated into the pane that didn't have the diff view originally.

GIF-ment:

![diff-copy](https://user-images.githubusercontent.com/734194/30403216-32f656a8-98a6-11e7-8575-328a7bc665b2.gif)

Can't reproduce on macOS 10.12.6 so tagging as Linux for now.

**Comment by rsese** [comment]
Thanks for moving this over from Discuss :+1:

I remember I had reproduced with `1.20.0-beta5` using <kbd>alt-tab</kbd> but I just tried to reproduce again with the latest beta `1.21.0-beta0` but couldn't reproduce anymore on Ubuntu 16.04. 

Can you confirm if you still see this in `1.21.0-beta0`?

**Comment by rsese** [comment]
Sorry tried again and can reproduce in `1.21.0-beta0` - I think when I tried laste time, I had double clicked on the Unstaged Changes tab for the file I changed (so the tab title wasn't italicized).  In that case, I can't reproduce.

If I just click an unstaged change in the Git panel and then click the other pane before <kbd>alt-tab</kbd>-ing, then I can reproduce:

![diff-copy](https://user-images.githubusercontent.com/734194/30403216-32f656a8-98a6-11e7-8575-328a7bc665b2.gif)

Thanks again for the report :bow:

**Comment by kuychaco** [comment]
Thanks for the report! This is a duplicate of #1163, and was closed in #1165. The fix is currently on dev and will reach beta/stable as part of v1.22

**Comment by rsese** [comment]
ahhh I missed that one, thanks @kuychaco :bow:
